public/index.php

```diff
@@ -4,12 +4,12 @@
 $minPHPVersion = '7.1';
 if (phpversion() < $minPHPVersion)
 {
-	die("You PHP version must be {$minPHPVersion} or higher to run CodeIgniter. Current version: ". phpversion());
+	die("You PHP version must be {$minPHPVersion} or higher to run CodeIgniter. Current version: " . phpversion());
```

Comment:
Picky, but "your PHP version..."

Reply:
Very true -that's what I get for copy pasting and not reviewing in  enough detail. :)